modules/tags.js

```diff
@@ -0,0 +1,25 @@
+var Tags = ``;
+
+function randomIcons(){
+    var icons = ["gallery_icon","channel_icon","video_icon","",""];
+    return icons[Math.floor(Math.random() * icons.length)];
+}
+
+for(var i = 11;i<20;i++){
```

Comment:
What is `i` and why is its default value `11`?

Reply:
i is the iterator.